src/components/engineService.ts

```diff
@@ -211,6 +214,30 @@ export class EngineService {
 				return args;
 			}
 
+			// execute single codemod
+			if (
+				'kind' in message.command &&
+				message.command.kind === 'executeCodemod'
+			) {
+				// @TODO engine should support such calls
```

Comment:
so this feature is not implemented yet right?

Reply:
yes, its not implemented, but i still can work on this feature and connect it to the node engine when its done.